content/es/es6.md

```diff
@@ -51,10 +53,21 @@ Puedes ver una lista más detallada, incluyendo la comparación con otros motore
 
 ## ¿Qué funcionales de ES6 están detrás de la --es_staging flag?
 
-* [Clases](https://github.com/lukehoban/es6features#classes) (sólo en strict mode)
-* [Object literal extensions](https://github.com/lukehoban/es6features#enhanced-object-literals)
+* [Clases](https://github.com/lukehoban/es6features#classes) (sólo en strict mode, utilizando la flag `--harmony_classes` lo que implica que el block scope y object literal extensions)
```

Comment:
I would replace:
"... lo que implica que el block scope ..." by "... lo que implica block scope ..."

Also, Do you think the expression for flag should be "la flag" o "el flag"?

Lastly, I think that if we replace "tras el/la flag" by "utilizando el/la flag", we should replace that expression in all the document :)